platform/darwin/docs/theme/assets/css/jazzy.css.scss

```diff
@@ -406,10 +438,7 @@ pre code {
   }
 }
 
-.item-container { }
-
 .item {
```

Comment:
The hit target for auto-expanding section links hugs the text too closely, making it difficult to click the link. Any chance we could increase the padding around these links, without necessarily spacing the links farther apart?

<img width="172" alt="link" src="https://cloud.githubusercontent.com/assets/1231218/21218644/43efaf5c-c266-11e6-8bd2-0a4dfbeab132.png">